orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/job/WaitOnJobCompletion.groovy

```diff
@@ -52,6 +53,23 @@ public class WaitOnJobCompletion extends AbstractCloudProviderAwareTask implemen
 
   static final String REFRESH_TYPE = "Job"
 
+  @Override
+  long getDynamicTimeout(@Nonnull StageExecution stage) {
+    JobTimeout timeout = stage.mapTo(JobTimeout.class)
+    if (timeout.timeoutMinutes.isPresent()) {
+      return TimeUnit.MINUTES.toMillis(timeout.timeoutMinutes.getAsLong())
```

Comment:
The [existing default timeout](https://github.com/spinnaker/orca/pull/3587/files#diff-75848b16a2174760f7f312a0b888d6c1R40) is in hours, but the dynamic one is in minutes. Everything gets converted to milliseconds, but if it makes sense, I can change the default to minutes as well, for better readability maybe?

Reply:
Up to you...I guess it might slightly increase readability, so might not be a bad idea.  One thing would be great regardless is to update that default timeout to be `final` as we should definitely never be changing that global value.

Reply:
Makes sense - will update that